appveyor.yml

```diff
@@ -18,7 +18,7 @@ environment:
     RUBY_VER: 24
   - PUPPET_GEM_VERSION: ~> 5.0
     RUBY_VER: 24-x64
-  - PUPPET_GEM_VERSION: 4.7.1
+  - PUPPET_GEM_VERSION: 4.9.0
```

Comment:
I think we need to change this in `.sync.yml` also, otherwise next modsync will blow it away, right?

https://github.com/puppetlabs/modulesync_configs/blob/master/config_defaults.yml#L123-L138

Will update the commit accordingly...

Reply:
This is PDKified.  There is no modsync now